architecture/core_concepts/projects_and_users.adoc

```diff
@@ -146,6 +146,28 @@ include any application objects for access by multiple projects, such as
 templates and images. These objects should be those that do not require
 communication between the pods.
 
-
-
-
+ifdef::openshift-online[]
+[[projects-idling]]
+== Project Idling
+In {product-title} Starter, a project that is inactive for more than 30 minutes
```

Comment:
30 minutes maybe need update.  Need double confirm with @sallyom .

Reply:
@ahardin-rh yes, that's currently 24 hrs! lol but it will decrease as unidling time improves.  Maybe 'for more than a configured duration of time'?